config/test.json

```diff
@@ -1,5 +1,6 @@
 {
   "features": {
+    "azureAuthEnabled": false,
```

Comment:
How come this is disabled in test?

Reply:
Because the config isn't there yet (in parameter store) so it'll just fail. If we can reach consensus on how the secrets should be configured here, I'll store them there and update this.

Reply:
👍